ios/Classes/TiMapView.m

```diff
@@ -1070,7 +1069,7 @@ - (TiMapAnnotationProxy *)proxyForAnnotation:(MKAnnotationView *)pinview
   for (id annotation in [map annotations]) {
     if ([annotation isKindOfClass:[TiMapAnnotationProxy class]]) {
       if ([(TiMapAnnotationProxy *)annotation tag] == pinview.tag) {
-        [annotation setView:pinview];
+        //[annotation setView:pinview];
```

Comment:
@anil-shukla-axway  Why this line commented out? It was introduced in PR #297 to fix https://jira.appcelerator.org/browse/MOD-2103.

Reply:
work is in progress on this, as due to this #297 now the changes done in my PR #293 are getting crashed. So I need to find an alternative way for MOD-2103.